coder_sniffer/Drupal/Sniffs/Classes/FullyQualifiedNamespaceSniff.php

```diff
@@ -127,6 +127,21 @@ public function process(File $phpcsFile, $stackPtr)
             $useStatement = $phpcsFile->findNext(T_USE, ($endPtr + 1));
         }//end while
 
+        $classStatement = $phpcsFile->findNext(T_CLASS, 0);
+        while ($classStatement !== false) {
```

Comment:
We should probably skip this conflict check if we already received a $conflict earlier in this sniff.